src/ophys_etl/transforms/sine_dewarp.py

```diff
@@ -0,0 +1,606 @@
+import numpy as np
+import os
+import h5py
+import math
+import time
+import json
+import argparse
+import logging
+import multiprocessing
+import functools
+
+
+# Default parameters
+minvalue = 0.0
+maxlevel = 65535  # 16bit
+
+valid_nr_methods = [0, 1, 2, 3]
+
+
+def mode(data: np.ndarray):
```

Comment:
I think this entire function is covered by `scipy.stats.mode`. Please confirm and replace.

Reply:
You're right, I'm adding that now